.gitignore

```diff
@@ -18,6 +18,14 @@ Gemfile.lock
 coverage
 pkg
 .internal_test_app
+.byebug_history
+
+# IDE files
+.idea
```

Comment:
All of these changes belongs in your local global gitignore, not in the project .gitignore

https://robots.thoughtbot.com/global-gitignore